db/migrate/20170415190055_add_extra_date_columns_to_calendar_dates.rb

```diff
@@ -0,0 +1,7 @@
+class AddExtraDateColumnsToCalendarDates < ActiveRecord::Migration[5.1]
+  def change
+    add_column :calendar_dates, :date_hour, :integer, required: true
+    add_column :calendar_dates, :date_minute, :integer, required: true
+    add_column :calendar_dates, :date_second, :integer, required: true
+  end
```

Comment:
I just read the method signature wrong...must have stopped reading at `format_` 🤦‍♂️